app/components/buttons/ToggleSwitch.js

```diff
@@ -0,0 +1,27 @@
+import "style/ToggleSwitch.less";
```

Comment:
Could you please change this with css module ? have a look `*.module.css` files and how we use them in `.jsx` files.